apps/cookbook/src/app/examples/empty-state-example/examples/message-types.ts

```diff
@@ -0,0 +1,38 @@
+import { Component } from '@angular/core';
+
+const config = {
+  template: `<kirby-empty-state
+  iconName="verify"
+  themeColor="success"
+  title="Success"
+  subtitle="Additional messaging via subtitle"
+></kirby-empty-state>
+<kirby-empty-state
+  iconName="help"
+  themeColor="warning"
+  title="Warning"
+  subtitle="Additional messaging via subtitle"
+></kirby-empty-state>
+<kirby-empty-state
+  iconName="overview-outline"
+  title="Empty"
+  subtitle="Additional messaging via subtitle"
+></kirby-empty-state>`,
+};
+
+@Component({
+  selector: 'cookbook-empty-state-message-types-example',
+  template: config.template,
+  styles: [
+    `
+      :host {
+        display: flex;
+        gap: 2rem;
```

Comment:
By using rem on gap (or other spacing properties like margin) you will get a larger distance between the elements when increasing font-size. Consider if this should be defined in pixels instead to have a static distance between items. I guess this is more of a conceptual consideration that we can also discuss later in more relevant issues. 😄

Reply:
My intention here was for the gap to grow a little bit when text becomes larger, to still have a nice visual separation on smaller screen widths.

But I reduced it a bit to 1 rem now 👍